include/cantera/thermo/IdealGasPhase.h

```diff
@@ -386,6 +386,22 @@ class IdealGasPhase: public ThermoPhase
         setDensity(p * meanMolecularWeight() / RT());
     }
 
+    //! Set the internally stored temperature of the phase (K).
+    //! Since the gas is in equilibrium, the electron temperature
+    //! is also set to the same value.
+    //! @param temp Temperature in Kelvin
```

Comment:
```suggestion
    //! @param temp  Temperature in Kelvin
```
I *think* that double spaces are commonly used after the parameter name in Cantera (here and elsewhere)